forms/gridfield/GridFieldAddExistingAutocompleter.php

```diff
@@ -280,38 +334,46 @@ public function getSearchFields() {
 
 	/**
 	 * Detect searchable fields and searchable relations.
-	 * Falls back to {@link DataObject->summaryFields()} if
-	 * no custom search fields are defined.
-	 * 
-	 * @param  String the class name
-	 * @return Array|null names of the searchable fields
+	 *
+	 * Falls back to {@link DataObject->summaryFields()} if no custom search fields are defined.
+	 *
+	 * @param string $dataClass
+	 *
+	 * @return null|array
 	 */
 	public function scaffoldSearchFields($dataClass) {
 		$obj = singleton($dataClass);
+
 		$fields = null;
+
 		if($fieldSpecs = $obj->searchableFields()) {
 			$customSearchableFields = $obj->stat('searchable_fields');
+
 			foreach($fieldSpecs as $name => $spec) {
 				if(is_array($spec) && array_key_exists('filter', $spec)) {
-					// The searchableFields() spec defaults to PartialMatch,
-					// so we need to check the original setting.
-					// If the field is defined $searchable_fields = array('MyField'),
-					// then default to StartsWith filter, which makes more sense in this context.
+					// The searchableFields() spec defaults to PartialMatch, so we need to check
+					// the original setting.
+
+					// If the field is defined $searchable_fields = array('MyField'), then default
+					// to StartsWith filter, which makes more sense in this context.
+
 					if(!$customSearchableFields || array_search($name, $customSearchableFields)) {
 						$filter = 'StartsWith';
 					} else {
 						$filter = preg_replace('/Filter$/', '', $spec['filter']);
 					}
-					$fields[] = "{$name}:{$filter}";
+
+					$fields[] = sprintf('%s:%s', $name, $filter);
```

Comment:
It feels like a lot of rewriting is going on that changes style without necessarily improving it. I'm just concerned that these changes will make merging up harder, since more rewriting has to be done when there are conflicts.